en/docs/edit-the-code/code-navigation.md

```diff
@@ -1 +1,31 @@
 # Code navigation
+
+## Go to definition 
+
+This feature navigates you to the definition of a particular symbol. For example, when you invoke the `Go To Definition` option on a function call expression, it navigates you to the definition of the function.
+
+![Go to definition support](/en/docs/img/edit-the-code/code-navigation/go-to-definition.gif?raw=true)
+
+## Go to symbol
+
+This feature allows you to navigate symbols inside a file.
+
+![Go to symbol](/en/docs/img/edit-the-code/code-navigation/go-to-symbol.gif?raw=true)
+
+## Find all references
+
+Invoking the references on a symbol will prompt you with all the symbol references in the current project.
+
+![Find all references](/en/docs/img/edit-the-code/code-navigation/find-all-references.png?raw=true)
+
+## Peek
+
+This feature allows you to peek the definition or references of a given symbol.
```

Comment:
What do we mean by peek?

https://code.visualstudio.com/docs/editor/editingevolved#_peek

Reply:
A TW will address this by referring the VS code docs.